tasks/vendoring/patches/vendor/pythonfinder-import.patch

```diff
@@ -0,0 +1,78 @@
+diff --git a/pipenv/vendor/pythonfinder/compat.py b/pipenv/vendor/pythonfinder/compat.py
```

Comment:
sorry again for the patches. For most of these they really should be caught by the vendoring import rewrites but for some reason they arent